hugegraph-test/src/main/java/org/apache/hugegraph/api/MetricsApiTest.java

```diff
@@ -234,8 +259,8 @@ public void testMetricsBackend() {
                     String key = (String) e.getKey();
                     value = e.getValue();
                     Assert.assertTrue(String.format(
-                                      "Expect map value for key %s but got %s",
-                                      key, value),
+                                              "Expect map value for key %s but got %s",
```

Comment:
prefer to keep the origin alignment

Reply:
seems done？ if we choose all commit
![image](https://github.com/apache/incubator-hugegraph/assets/48077841/bd8d8020-5e47-4b5b-922e-13bae6ff912a)